README.md

```diff
@@ -1,33 +1,93 @@
+
+### More about Sendbird SyncManager for Android
+
+Find out more about Sendbird SyncManager for Android on [SyncManager for Android doc](https://sendbird.com/docs/syncmanager/v1/android/getting-started/about-syncmanager). If you have any comments or questions regarding bugs and feature requests, visit [Sendbird community](https://community.sendbird.com). 
+
+<br />
+
+## Before getting started
+
+This section shows the prerequisites you need to check to use Sendbird SyncManager for Android.
+
+### Requirements 
+
+The minimum requirements for SyncManager for Android are:
+
+- `Android 4.0 (API level 14) or higher`
+- `Java 7 or higher`
+- `Gradle 3.4.0 or higher`
+
+<br />
+
+## Getting started
+
+This section gives you information you need to get started with Sendbird SyncManager for Android. 
+
+### Try the sample app
+
+Download the sample app to test the core features of SyncManager for Android. 
+
+- https://github.com/sendbird/SendBird-Android/tree/master/syncmanager
+
+> **Note**: The fastest way to test our SyncManager is to build your chat app on top of our sample app. Make sure to change the application ID of the sample app to your own. Go to the [Create a Sendbird application from your dashboard section](#step-1-create-a-sendbird-application-from-your-dashboard) to learn more.
+
+### Step 1: Create a Sendbird application from your dashboard
+
+A Sendbird application comprises everything required in a chat service including users, message, and channels. To create an application:
+
+1. Go to the Sendbird Dashboard and enter your email and password, and create a new account. You can also sign up with a Google account.
+2. When prompted by the setup wizard, enter your organization information to manage Sendbird applications.
+3. Lastly, when your dashboard home appears after completing setup, click **Create +** at the top-right corner.
+
+Regardless of the platform, only one Sendbird application can be integrated per app; however, the application supports communication across all Sendbird’s provided platforms without any additional setup. 
+
+> **Note**: All the data is limited to the scope of a single application, thus users in different Sendbird applications are unable to chat with each other. 
+
+### Step 2: Download SyncManager for Android
+
+Sendbird SyncManager currently supports iOS, Android, and JavaScript SDKs. You can download SyncManager for Android from our repository on GitHub.
+
+### Step 3: Install SyncManager for Android
+
+SyncManager for Android is installed through ‘Gradle’.
+
+```gradle
 repositories {
     maven { url "https://raw.githubusercontent.com/sendbird/sendbird-syncmanager-android/master/" }
 }
 dependencies {
-    // SyncManager
-    implementation 'com.sendbird.sdk:sendbird-syncmanager:1.1.28'
+    // SyncManager SDK for Android (Latest, embeds sendbird sdk 3.0.147)
+    implementation 'com.sendbird.sdk:sendbird-syncmanager:1.1.26'
```

Comment:
The latest version is 1.1.28, and it embeds sendbird sdk 3.0.150)

Reply:
Done!